pkg/operator/starter.go

```diff
@@ -128,22 +128,28 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		ctx.EventRecorder,
 	)
 
-	certRotationController, err := certrotationcontroller.NewCertRotationController(kubeClient, operatorClient, kubeInformersForNamespaces, ctx.EventRecorder)
+	certRotationController, err := certrotationcontroller.NewCertRotationController(
+		kubeClient.CoreV1(),
```

Comment:
we want to use a cached one here, right?

Reply:
yes, when library-go portion merge and I get it in bump